Dockerfile

```diff
@@ -1,36 +1,186 @@
-FROM quay.io/refgenomics/docker-scipy-notebook:v0.1.6
-MAINTAINER Nick <nick@onecodex.com>
-
-# Add jessie-backports repository, install bwa 0.7.13 and samtools 1.3.1
-USER 0
-RUN echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/bioinformatics.list \
-    && apt-get -y update \
-    && apt-get -t jessie-backports install -y "samtools" \
-    && apt-get -t jessie-backports install -y "bwa" \
-    && apt-get -t jessie-backports install -y libffi-dev libcairo2-dev libpango1.0-dev  \
-    && apt-get clean
+# Copyright (c) Reference Genomics, Inc.
+# Distributed under the terms of the Modified BSD License.
+# Extended from github.com/jupyter/docker-stacks
+# See also http://blog.dscpl.com.au/2016/01/roundup-of-docker-issues-when-hosting.html
 
-# Add assets
-RUN mkdir /opt/onecodex/
-ADD install/* /opt/onecodex/
-RUN chmod +x /opt/onecodex/notebook_report.py && \
-    ln -s /opt/onecodex/notebook_report.py /usr/local/bin/notebook_report.py
+FROM python:3.6.6-slim-stretch
+
+MAINTAINER Nick Greenfield <nick@onecodex.com>
+
+USER root
+
+# Install all OS dependencies for notebook server that starts but lacks all
+# features (e.g., download as all possible file formats)
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+    wget \
+    bzip2 \
+    ca-certificates \
+    gnupg \
+    apt-transport-https \
+    sudo \
+    locales \
+    git \
+    vim \
+    build-essential \
+    python-dev \
+    unzip \
+    fonts-dejavu \
+    gfortran \
+    gcc \
+    cmake \
+    curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-USER 1000
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+    locale-gen
+
+# Install Tini
+RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.9.0/tini && \
```

Comment:
Can we update to `v0.18.0`? That's the [latest now](https://github.com/krallin/tini/releases).

Reply:
For sure, I'll check it out.

Reply:
Updated, didn't seem to cause any issues.